roles/manage-jira/tasks/create_project.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create Sample Project
+  uri:
+    url: "{{ atlassian.jira.url }}/rest/api/2/project"
+    method: POST
+    user: "{{ atlassian.jira.username }}"
+    password: "{{ atlassian.jira.password }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+    header:
+      - Accept: 'application/json'
+      - Content-Type: 'application/json'
+    body: "{ 'key': '{{ atlassian.jira.key }}',
+             'name': '{{ atlassian.jira.project }}',
+             'projectTypeKey': '{{ atlassian.projectTypeKey | default('software') }}',
+             'projectTemplateKey': '{{ atlassian.projectTemplateKey | default('com.pyxis.greenhopper.jira:gh-simplified-scrum') }}',
+             'description': 'Project created using Ansible playbook',
+             'lead': '{{ atlassian.jira.username }}',
```

Comment:
This should probably be it's own fact - i.e.: something like `atlassian.jira.lead` ... maybe you can default it to `username` if the `lead` isn't set